src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -418,4 +420,22 @@ public void receivePowersModified() {
             }
         }
     }
+
+    @Override
+    public void receiveStartAct() {
+        for (AbstractCard c : AbstractDungeon.player.masterDeck.group) {
+            if (c instanceof StartActSubscriber) {
+                ((StartActSubscriber)c).receiveStartAct();
```

Comment:
I think forwarding the `StartGameSubscriber` and `StartActSubscriber` notifications is a perfectly reasonable design for Equinox. However, I would not recommend forwarding this notification to *every* master deck card that implements  the BaseMod subscribers like this, because it would be possible in theory for other mods to exist that have cards of their own which implement the BaseMod subscribers and have their own separate mechanisms for forwarding the same events, and we don't want to cause such a mod to receive 2 copies of those events.

I think having `JorbsMod` implement BaseMod's `receiveStartAct` and `receiveStartGame` is good, but I'd suggest not using the BaseMod interfaces for the card; instead, I'd recommend adding equivalent methods on `CustomJorbsModCard` (with no-op default behavior), and only have the `JorbsMod` hooks invoke the `CustomJorbsModCard` hooks.